src/browserlib/get-code-elements.mjs

```diff
@@ -0,0 +1,22 @@
+import informativeSelector from './informative-selector.mjs';
+import cloneAndClean from './clone-and-clean.mjs';
+
+/**
+ * Helper function that returns a set of code elements in document order based
+ * on a given set of selectors, excluding elements that are within an index.
+ *
+ * The function excludes elements defined in informative sections.
+ *
+ * The code elements are cloned and cleaned before they are returned to strip
+ * annotations and other asides.
+ */
+export default function getCodeElements(codeSelectors, excludeSelectors) {
```

Comment:
I think the code might be a bit easier to parse if excludeSelectors was part of an option object (à la `getCodeElements(selectors, {excludeSelectors: []})`)